src/scheduler/Makefile.am

```diff
@@ -135,3 +135,6 @@ dist_sysconf_DATA = \
 	pbs_resource_group \
 	pbs_sched_config
 
+if UNDOLR_ENABLED
+common_libs += @libundolr_lib@
```

Comment:
@riyazhakki If I am not wrong then this should be `pbs_sched_LDADD += @libundolr_lib@` right?

Reply:
@hirenvadalia thanks for catching this, i have raised the PR #1403 with one line change, please review and provide your approval. thank you.